src/index.js

```diff
@@ -0,0 +1,46 @@
+import './style.css';
+import './logo.png';
+import {
+  fetchCategoryMeals, displayMeals, transferLikes, likesCountUpdate,
+} from './modules/renderList.js';
+import { updateCategoryCounts } from './modules/mealCounter.js';
+
+const menuLinks = document.querySelectorAll('header a');
+const containerMeals = document.getElementById('list');
+
+const renderMealCategory = async (category) => {
+  const categoryMeals = await fetchCategoryMeals(category);
+  displayMeals(categoryMeals);
+};
+
+const setupHeaderLinkListeners = () => {
+  menuLinks.forEach((link) => {
+    const { dataset: { category } } = link;
+    link.addEventListener('click', async (e) => {
+      e.preventDefault();
+      await renderMealCategory(category);
+    });
+  });
```

Comment:
- [ ] when trying to add a comment to a meal the user receives the below error, please make sure to fix this before requesting another review.

![image](https://github.com/MohammadYaser/Javascript-capstone/assets/39240820/f3b80d79-9495-4a85-9097-8d472b9c4c1b)

Reply:
Hi @mikethreels 
we both have checked the problem you have mentioned but we didn't find any issue like this